cfaster/scrapers/utils/urls.py

```diff
@@ -3,15 +3,48 @@
 
 logger = logging.getLogger(__name__)
 
+def url_filter(string, to_remove):
+    if string.startswith(to_remove):
+        return string[len(to_remove):]
+    else:
+        return string
```

Comment:
It would be interesting doing it recursively.
`while strings.startswith(to_remove):`
Inputs like `https://https://google.com` would be treated correctly.

Reply:
That's not a valid URL to begin with-- what do you think the function should do?

`validate_url('an invalid url that ends with google.com', 'google.com')`-- I'm thinking that should return `False`

Reply:
It is not a valid URL but since the user is the one making the input we have to consider all test cases. If he uses `https://https://bla.com` and we remove only one of the `https://` we get `https://bla.com` which breaks our logic. I think it breaks when it splits the URL using `/` as the delimiter.
Not sure if the right decision is accepting invalid inputs and treating them (HTML does it a lot) or sending an error to the user, I think a regex following an RFC specification of how URLs should be is interesting.